engine/src/main/java/com/redhat/cloud/notifications/processors/camel/CamelTypeProcessor.java

```diff
@@ -41,6 +41,7 @@
 import static com.redhat.cloud.notifications.models.NotificationHistory.getHistoryStub;
 import static java.nio.charset.StandardCharsets.UTF_8;
 
+// TODO NOTIF-603 Migrate to orgId (not started in this class)
 @ApplicationScoped
 public class CamelTypeProcessor implements EndpointTypeProcessor {
```

Comment:
This will be done in a separate PR. It requires changes in our integration with RHOSE.

Reply:
https://issues.redhat.com/browse/NOTIF-706